src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -279,10 +282,13 @@ void drawing_classifier::init_training(
   // Read user-specified options.
   init_options(opts);
 
+  add_or_update_state({
+      {"training_iterations", 0},
+  });
+
   if (read_state<flexible_type>("random_seed") == FLEX_UNDEFINED) {
-    std::random_device random_device;
-    int random_seed = static_cast<int>(random_device());
-    add_or_update_state({{"random_seed", random_seed}});
+    std::random_device rd;
+    add_or_update_state({{"random_seed", (int)rd()}});
```

Comment:
Why is this being changed? The old implementation looked cleaner to mw

Reply:
sure, I will put it back. `rd` is often used name, though.

Reply:
Well, the previous one cannot compile.